OLSE.R

```diff
@@ -0,0 +1,17 @@
+z=read.csv("C:\\Users\\edaad\\Downloads\\dataunina.csv",header=T)

```

Comment:
it would be great if you could set it to a relative path so that I can run it immediately on my computer